softperipheral/doc/sQSPI/nrf54H20_porting_v1_0_0.rst

```diff
@@ -226,17 +227,28 @@ RAM configuration
 *****************
 
 The sQSPI Soft Peripheral operates from RAM.
+
+.. note::
+   Starting from sQSPI 1.0.0, Position Independent Code (PIC) is supported.
+   This allows an application to determine where to load the Soft Peripheral firmware.
+   The start address default value is defined in the :file:`nrf_config_sqspi.h` file but you can override it, for example, in :file:`nrfx_config.h`.
```

Comment:
It should be clarified somewhere around here if any code position is considered "supported" - tested well enough, or do we consider only selected one or two positions supported and others are "experimental"